src/Biomass/FarmValuesTable.js

```diff
@@ -118,6 +118,20 @@ const FarmValuesTable = ({
         percent_cellulose: rec.percent_cellulose || "N/A",
         percent_lignin: rec.percent_lignin || "N/A",
       }))
+      .filter((data) => {
+        if (
+          data.additional_contact === "-999" ||
+          data.county === "-999" ||
+          data.email === "-999" ||
+          data.address === "-999" ||
+          data.latlng === "-999" ||
+          data.notes === "-999" ||
+          data.state === "-999" ||
+          data.protocols_enrolled === "-999"
```

Comment:
This filtering is wrong. I want to ONLY filter out `protocols_enrolled === "-999"`. The others are artifacts that I want to have NOT be changed on unenroll